src/mmrelay/plugin_loader.py

```
@@ -1120,21 +1140,32 @@ def _run_git(
     return _run(cmd, timeout=timeout, **kwargs)
 
 
-def _check_auto_install_enabled(config: Any) -> bool:
+def _check_auto_install_enabled(
+    config: Any, plugin_type: str = PLUGIN_TYPE_CUSTOM
```

Ruff (0.15.10)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `config`

(ANN401)